pilot/pkg/features/pilot.go

```diff
@@ -191,6 +191,14 @@ var (
 		"PILOT_SIDECAR_USE_REMOTE_ADDRESS",
 		false,
 		"UseRemoteAddress sets useRemoteAddress to true for side car outbound listeners.")
+
+	// UseIstioJWTFilter enables to use Istio JWT filter as a fall back. Pilot injects the Istio JWT
```

Comment:
Would a user ever want the Istio JWT? Or is it just a fallback in case something breaks?

Reply:
(just curious, the flag is fine)

Reply:
I think we don't need this. We need to make sure the new JWT works before release. Having this fallback could make user supporting job harder.

Reply:
I think it makes things easier. If the envoy one breaks we can tell users to toggle this flag. Without the flag we have to scramble to get a fix out.

Its off by default and configurable in an obscure area so I don't think we would have users actively set this unless something was broken

Reply:
@howardjohn The fallback is just in case something breaks.

@diemtvu Do you prefer to delete the old code in this PR? I'm thinking to defer this until we're more confident the new filter will be working as expected (need more manual test and e2e tests). If we could finish all these test before 1.3 release, then I think could just remove them in a follow PR.

Reply:
Ok to keep this for now to be on the safe side. but may be add an issue to remove it for tracking and any confusion down the line.